client/helpers/parts.ts

```diff
@@ -56,7 +56,7 @@ export interface CreatePartFromFileRes {
   readonly queued?: QueuedTranslation;
 }
 
-interface QueuedTranslation {
```

Comment:
These are referenced by other objects in the doc so need to be exported so they also exist.